htdocs/api/class/api_status.class.php

```diff
@@ -43,14 +45,23 @@ public function __construct()
 	 */
 	public function index()
 	{
-		global $conf;
+		global $dolibarr_main_prod;
 
-		return array(
+		$response = array(
 			'success' => array(
 				'code' => 200,
 				'dolibarr_version' => DOL_VERSION,
-				'access_locked' => (!getDolGlobalString('MAIN_ONLY_LOGIN_ALLOWED') ? '0' : $conf->global->MAIN_ONLY_LOGIN_ALLOWED),
+				'access_locked' => (!getDolGlobalString('MAIN_ONLY_LOGIN_ALLOWED') ? '0' : getDolGlobalString('MAIN_ONLY_LOGIN_ALLOWED')),
 			),
 		);
+
+		if ( $dolibarr_main_prod == '0' ) {
+			$response['success']['environment']       = 'non-production';
+			$response['success']['timestamp_now_utc'] = dol_now();
+			$response['success']['timestamp_php_tz']  = date_default_timezone_get();
+			$response['success']['date_tz']           = date('Y-m-d H:i:s');
```

Comment:
Use of date() is forbidden. Can you use instead dol_print_date(dol_now('gmt'), ...

Reply:
done + force-pushed